source/Grid/Grid.js

```diff
@@ -1125,20 +1126,32 @@ export default class Grid extends PureComponent {
   }
 
   /**
+   * Check if the difference between current time and the last scroll ended event is greater.
+   * than the scrollingResetTimeInterval prop, else schedule this function to execute again.
+   */
+  _delayScrollEnded = () => {
+    const { scrollingResetTimeInterval } = this.props;
+    if (Date.now() - this._scrollDebounceStart >= scrollingResetTimeInterval) {
+      this._debounceScrollEndedCallback();
```

Comment:
Yes, I agree that the pointer must be cleared, but this hasn't been doing on `_debounceScrollEndedCallback` function?